packages/bot-web-ui/src/stores/quick-strategy-store.js

```diff
@@ -66,17 +76,25 @@ export default class QuickStrategyStore {
 
     @action.bound
     setSelectedDurationUnit(duration_unit) {
-        this.selected_duration_unit = duration_unit;
+        this.qs_cache.selected_duration_unit = this.selected_duration_unit = duration_unit;
+        storeSetting('quick_strategy', this.qs_cache);
     }
 
     @action.bound
     setSelectedSymbol(symbol) {
-        this.selected_symbol = symbol;
+        this.qs_cache.selected_symbol = this.selected_symbol = symbol;
+        delete this.qs_cache.selected_duration_unit;
+        delete this.qs_cache.duration_value;
+        delete this.qs_cache.selected_trade_type;
```

Comment:
Do we need to delete these values? Doesn't the `storeSetting` call below just overwrite these values?

Reply:
deleting the qs_cache for changing symbol and type seems necessary because some of the symbols have different type of trade and duration unit if we don't delete cache it gets wrong results
e.g: if your symbol initially set AUD the trade type has 2 different options and have 5option for the duration unit but if you choose the Bull market index and have the cache for duration and trade type it just return 2 initial trade types and doesn't return Digits trade types. same for Duration units

Reply:
That makes sense, thank you for explaining. Let's keep it 👍